server/config.ts

```diff
@@ -49,6 +49,7 @@ export default {
   staticResourceCacheDuration: 20,
   flags: {
     oasysDisabled: process.env.OASYS_DISABLED || false,
+    voidsDisabled: !['local', 'dev', 'test'].includes(process.env.ENVIRONMENT || 'local'),
```

Comment:
Suggest pull out `const environment = process.env.ENVIRONMENT || 'local'`, and use that here and on line 54

Reply:
good call